lib/taxjar/api/customer.rb

```diff
@@ -0,0 +1,30 @@
+require 'taxjar/api/utils'
+
+module Taxjar
+  module API
+    module Customer
+      include Taxjar::API::Utils
+
+      def list_customers(options = {})
+        perform_get_with_array("/v2/customers", 'customers', options)
```

Comment:
at some point can probably take set the "/v2/customers" prefix somewhere or have it build off "#{version}/#{module_name/resource_name/whatever you wanna call it}"